docs/kintsugi/overview.md

```diff
@@ -2,6 +2,8 @@
 
 **Kintsugi** is interBTC's canary network, developed by Interlay under the supervision of Kintsugi Labs, a non-profit oriented organization responsible for Kintsugi's launch and the support of the decentralized network and ecosystem.
 
+The current Working Paper titled *Token Economy* is available [here](https://github.com/interlay/whitepapers/raw/master/Kintsugi_Token_Economy.pdf).
```

Comment:
This is already linked below see line 25. No need for this line.

Reply:
Ok, removed.
However I would also like to note that they are 3 broken links here, on lines 11, 17 and 23.